nextflow_schema.json

```
@@ -340,18 +340,21 @@
                 },
                 "hostremoval_reference": {
                     "type": "string",
+                    "format": "file-path",
```

```suggestion
                    "format": "file-path",
                    "exists": true
```
Can you also add this to all other files where it makes sense to check for existence?

---

Done